pages/blog/better-type-safety-for-resolvers-with-graphql-codegen.mdx

```diff
@@ -77,12 +79,18 @@ Now, TypeScript engine will verify that object you returned, and you'll be able
 
 Also, if you'll change your schema types and re-run the codegen (or use [Watch Mode](https://graphql-code-generator.com/docs/getting-started/development-workflow#watch-mode)), it will re-generate the matching types and check your code again.
 
-As your probably understood, the default behaviour of `typescript-resolvers` is using the base type generated by `typescript`, that means, that your schema types and resolvers needs to match and have the same signature and structure.
+As your probably understood, the default behavior of `typescript-resolvers` is using the base type generated by `typescript`, that means, that your schema types and resolvers needs to match and have the same signature and structure.
 
 But it's not always the case - because your GraphQL schema, in most cases, isn't the same as your models types - this is why we have `mappers` configuration.
 
 ## Use Your Model Types
 
+Models types are the way your data is being stored or represented behind the scenes. Think about `User` object from example - in most cases, the representation of `User` in your database (or any other downstream API) is different than the way your represent `User` in your API. Sometimes it's because of security considerations, and sometimes because fields are internal and used only by you, and not by the consumers.
```

Comment:
I thought to even mention some tools that could give you types from your Models, but maybe it's not necessary and also too broad because many times you create those yourself?

Tools like Swagger generator, gRPC SDKs, pgTyped and GraphQL Mesh